eox_nelp/utils.py

```diff
@@ -2,7 +2,12 @@
 import re
 from copy import copy
 
+from opaque_keys.edx.keys import CourseKey
+
+from eox_nelp.edxapp_wrapper.course_overviews import CourseOverview
+
 NATIONAL_ID_REGEX = r"^[1-2]\d{9}$"
+COURSE_ID_REGEX = r'(course-v1:[^/+]+(/|\+)[^/+]+(/|\+)[^/?]+)'
```

Comment:
Where was extracted this Regex?
It seems a little different than 
https://github.com/openedx/edx-platform/blob/master/openedx/core/constants.py#L12
```python
>>> from django.conf import settings
>>> settings.COURSE_KEY_REGEX
'(?:[^/+]+(/|\\+)[^/+]+(/|\\+)[^/?]+)'
```

Reply:
from that place the modification is the course-v1: